MaterialLib/PhysicalConstant.h

```diff
@@ -86,5 +93,18 @@ namespace SpecificGasConstant
 /// Specific gas constant for water vapour.
 const double WaterVapour = IdealGasConstant / MolarMass::Water;  // = 461.504;
 }  // namespace SpecificGasConstant
+/**
+* Henry's law constant
+* It is defined here as the ratio of the aqueous-phase concentration of a
+* chemical to its equilibrium partial pressure in the gas phase.
+*/
+namespace HenryConstant
+{
+/**Henry constant for hydrogen, Please refer to
+* De Nevers N. Physical and chemical equilibrium for chemical engineers[M].
+* John Wiley & Sons, 2012.
+*/
+double const HenryConstH2 = 7.65e-6;  /// mol/Pa./m3
```

Comment:
Either write HenryConstantH2 or drop the HenryConst (it is already indicated by the namespace).